src/commands/wait.yml

```
@@ -8,9 +8,9 @@ steps:
       name: Wait for LocalStack to be ready
       command: |
         for i in {1..45}; do
-          if [ `docker logs localstack_main | grep 'Ready.'` ]; then
+          if [ `docker logs localstack-main | grep 'Ready.'` ]; then
             break
           fi
```

Here we could also just check the old container name as a fallback to implement backwards-compatibility:
```suggestion
          if [ `docker logs localstack-main | grep 'Ready.'` ]; then
            break
          fi
          if [ `docker logs localstack_main | grep 'Ready.'` ]; then
            break
          fi
```